server/export.js

```diff
@@ -80,6 +80,11 @@ export default async function (dir, options) {
     hotReloader: null
   }
 
+  // Static hosted sites sometimes have different url requirements
+  if (config.exportAssetPrefix) {
+    renderOpts.assetPrefix = config.exportAssetPrefix.replace(/\/$/, '')
```

Comment:
I see what you're pointing out though with the production check in the config file. does the config function expose any context regarding whether static export is happening or would I need to run my command with NODE_ENV=production? Also I'm good with closing this.